docs/topics/gradle-best-practices.md

```diff
@@ -0,0 +1,141 @@
+[//]: # (title: Gradle best practices)
+
```

Comment:
If a newbie cold lands on this page it'd be nice to give them a brief sentence to help them understand what this Gradle thing is.

E.g. something like this:

```suggestion
[//]: # (title: Gradle best practices)

[Gradle](https://gradle.org/) is a tool used by many Kotlin projects to build the projects.
```

(Also, add a clickable link to the Gradle homepage if they want to deep dive.)